docker-compose.yml

```diff
@@ -3,63 +3,67 @@ version: "3"
 services:
   emintdnode0:
     container_name: emintdnode0
-    image: "emintd/node"
```

Comment:
I'd suggest keeping the image names as they are now to isolate the changes

Reply:
sure, but we would need to change it everywhere on the makefile as well, that can be an easy change